src/api/data/cache/charthelper/icon_helper.go

```diff
@@ -0,0 +1,142 @@
+package charthelper
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/disintegration/imaging"
+	"github.com/helm/monocular/src/api/swagger/models"
+)
+
+const originalFormat = "original"
+
+type iconFormat struct {
+	name   string
+	width  int
+	height int
+}
+
+var availableFormats = []iconFormat{
```

Comment:
@Angelmmiguel it makes a lot of sense, thanks a lot!

To reduce computation penalty I'd like to reduce the number of processed icons to the minimum. What's in your take the formats (width, height and strategy) that we will need?